python/src/robyn/modeling/entities/enums.py

```diff
@@ -27,3 +31,6 @@ class Models(Enum):
 
     RIDGE = auto()  # Ridge Regression
     # Add other model types as needed
+
+
+HYPERPARAMETER_NAMES = ("thetas", "shapes", "scales", "alphas", "gammas", "penalty")
```

Comment:
Can you define formal Enum for this? What you currently have is Constant more than Enum

Reply:
I actually need is a constant not an enum (this is supposed to be used multiple files but many places has it implemented individually. Want it in a different file called constants?